gno.land/pkg/gnoweb/components/view_directory.go

```diff
@@ -6,8 +6,54 @@ type DirData struct {
 	PkgPath     string
 	Files       []string
 	FileCounter int
+	FilesLinks  FilesLinks
+	LinkType    DirLinkType
+	HeaderData  HeaderData
```

Comment:
Do you really need this ? I don't think this make sense here.

Reply:
I juste used this one HeaderData temporary for the template. The other are needed to understand the context and update the link depending on the dir type (namespace dir / folder dir). we can deduplicate that in the next UX PR regarding dir layout.

Reply:
It doesn't seems to be used anywhere